### PR TITLE
Update instruction for "Install Dependencies"

### PR DESCRIPTION
**Pull request by PikachuEXE**
This is to update `npm > yarn` switch for package installation step in
https://github.com/FreeTubeApp/FreeTube/pull/1692

This should **not** be merged until the above PR is merged

### CODE REVIEW

**Review by efb4f5ff-1298-471a-8973-3d47447115dc — APPROVED** [review]

**Comment by PikachuEXE** [comment]
The related PR is now merged so this PR can be merged now

**Review by peepopoggers — DISMISSED** [review]

File: development/getting-started.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ cd FreeTube
 Install Dependencies:
 
 ```
-npm install
+yarn install
```
Comment (jasonhenriquez) [inline development/getting-started.md:25]: Isn't it just `yarn`?

**Review by jasonhenriquez — CHANGES_REQUESTED** [review]

File: development/getting-started.md (hunk 1 above)
Comment (PikachuEXE) [reply development/getting-started.md:25]: Fine I will include both -,-

**Comment by PikachuEXE** [comment]
Updated!

**Review by Hiers — APPROVED** [review]
Instructions worked on my machine, seems good.

**Review by peepopoggers — APPROVED** [review]
Looking yarn good :+1:

**Review by jasonhenriquez — APPROVED** [review]
Nice job stitching this together!

**Pull request merged**